ports/swig/portfile.cmake

```
@@ -0,0 +1,28 @@
+set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO swig/swig 
+    REF "v${VERSION}"
+    SHA512 67c0a7b77563e56f93df95293d1a40dd74514f484087b146e48525a58de0f437c19b94669c796119edcb56268ea18ced7615163fd9a9a5b0dfe748c3d0028a4f
+    HEAD_REF master
+)
+
+vcpkg_find_acquire_program(BISON)
+get_filename_component(BISON_EXE_PATH "${BISON}" DIRECTORY)
+vcpkg_add_to_path("${BISON_EXE_PATH}")
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)

```

An approach without cluttering the path

```suggestion
vcpkg_find_acquire_program(BISON)

list(APPEND OPTIONS "-D BISON_EXECUTABLE=${BISON}")

vcpkg_cmake_configure(
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
      ${OPTIONS}
)
```

---

Thank you for all your suggestions!